Source/CarthageKit/Project.swift

```diff
@@ -626,6 +626,13 @@ public final class Project {
 					.map { project, _ in project }
 					.filter { project in dependenciesToInclude?.contains(project.name) ?? false })
 
+				// If the user has entered a dependency that isn't found in the Cartfile,
+				// the dependenciesToInclude array will exist and be non-empty, 
+				// and the projectsToInclude array will be empty. In this case, we need to return an error.
```

Comment:
It seems like this would only be true if you only included misspelled projects. (And not if, e.g., you spelled one correctly and one incorrectly.)